components/request_otr/resources/request_otr_interstitial.html

```diff
@@ -0,0 +1,36 @@
+﻿<!doctype html>
+<head>
+  <meta charset="utf-8">
+  <meta name="theme-color" content="#fff">
+  <title>$i18n{tabTitle}</title>
+  <link rel="stylesheet" href="../../../../components/security_interstitials/core/common/resources/interstitial_core.css">
+  <link rel="stylesheet" href="../../../../components/security_interstitials/core/common/resources/interstitial_common.css">
+  <link rel="stylesheet" href="request_otr_interstitial.css">
+  <script src="../../../../ui/webui/resources/js/util_deprecated.js"></script>
```

Comment:
Shouldn't this be `chrome://resources/js/util_deprecated.js`. In fact since that's deprecated, we should import from util_ts.ts.
In your .ts file you would put at the top:
```
import { $ } from 'chrome://resources/js/util_ts.js'
```

Reply:
After discussion in Slack I can't make this work, and no other interstitials do this, so I'm leaving it.